iothub_client/samples/pnp/readme.md

```diff
@@ -0,0 +1,23 @@
+# Samples to demonstrate Azure IoT Plug and Play
+
+The samples in this directory demonstrate how to implement a Azure IoT Plug and Play device.  Azure IoT Plug and Play is documented [here](aka.ms/iotpnp).  The samples assume basic familairity with PnP concepts, though not in depth knowledge of the PnP "convention".  The "convention" is a set of rules for serializing and deserialing data that uses  IoTHub primitives for transport.  The samples themselves demonstrate and have a sample helper library.
```

Comment:
```suggestion
The samples in this directory demonstrate how to implement an Azure IoT Plug and Play device.  Azure IoT Plug and Play is documented [here](aka.ms/iotpnp).  The samples assume basic familiarity with PnP concepts, though not in depth knowledge of the PnP "convention".  The "convention" is a set of rules for serializing and de-serializing data that uses IoTHub primitives for transport.  The samples themselves demonstrate and have a sample helper library.
```

Reply:
Could you clarify the last sentence above too?

Reply:
took changes && clarified.